pkg/os/terminal/terminal.go

```diff
@@ -0,0 +1,21 @@
+package terminal
+
+import (
+	"os"
+
+	xterminal "golang.org/x/term"
+)
+
+var (
+	// Global variable to force output regardless if terninal
+	ForceShowOutput bool = false
+)
+
+func IsShowTerminalOutput() bool {
```

Comment:
`IsShowTerminalOutput` does not read great. Just `ShowTerminalOutput`? or `ShouldShowTerminalOutput`?

Reply:
Merely following he `bool` pattern to prefix with `Is`. Minor concern.